modules/@angular/compiler/src/schema/dom_element_schema_registry.ts

```diff
@@ -254,6 +255,12 @@ export class DomElementSchemaRegistry implements ElementSchemaRegistry {
     }
   }
 
+  securityContext(tagName: string, propName: string): SecurityContext {
+    if (tagName === 'a' && propName === 'href') return SecurityContext.URL;
```

Comment:
I think we need to refactor this:
`SecurityContext.HTML` and `SecurityContext.URL` only make sense in a browser. I.e. you calculate them in the `DomElementSchemaRegistry'. Then the escaping for them should also be in the `DomRenderer`.